cogboard-webapp/src/components/WidgetTypeIcon.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import StatusIcon from './StatusIcon';
+import { StyledStatusIconButton, StyledIconWrapper } from './Widget/styled';
+
+const WidgetTypeIcon = props => {
```

Comment:
Please destructure props and use early return with if statement. It is more clean and readable than ternary.